### PR TITLE
JENKINS-36481# Rollbar and it.pluginVersion() removed

### PR DESCRIPTION
**Pull request by vivek**
**Decription**

**Submitter checklist**
- [x] Change is code complete and matches issue description
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

Check the footer of dashboard, should show commitId instead of version

**Reviewer checklist**
- [x] Run the changes and verified the change matches the issue description
- [x] Reviewed the code
- [x] Verified that the appropriate tests have been written or valid explaination given

@reviewbybees 

BlueOceanUI is a bootstrap and core infrastructure class, we do want to
keep it fiarly abstract and indepedent of direct Jenkins dependency.

Rollbar or any such analytics tool integration should be done out of blueocean-web
possibly inside a separate plugin or some place else that makes sense.

Footer version to display the commitId/branch

### CODE REVIEW

File: blueocean-web/src/main/resources/io/jenkins/blueocean/BlueOceanUI/index.jelly [hunk 1]
```diff
@@ -30,37 +29,6 @@
             <link rel="stylesheet"
                   href="${resURL}/plugin/blueocean-web/assets/css/jenkins-design-language.css"
                   type="text/css"/>
-
-            <j:if test="${it.includeRollbar()}">
```
Comment (kzantow) [inline blueocean-web/src/main/resources/io/jenkins/blueocean/BlueOceanUI/index.jelly]: 👍 👍

File: blueocean-web/src/main/js/DevelopmentFooter.jsx [hunk 2]
```diff
@@ -5,14 +5,6 @@ import revisionInfo from '../../../target/classes/io/jenkins/blueocean/revisionI
 
 export class DevelopmentFooter extends Component {
     render() {
-        if (!revisionInfo || !revisionInfo.name) {
-            var blueOceanVersion = document.getElementsByTagName('head')[0].getAttribute('data-blue-ocean-version');
-            return (
-                <div className="development-footer">
-                    <span>Blue Ocean UI v{blueOceanVersion}</span>
```
Comment (kzantow) [inline blueocean-web/src/main/js/DevelopmentFooter.jsx]: But now we won't see the revision number on screen, right? From the ticket, am I reading right that there will be a follow-on to restore it?

File: blueocean-web/src/main/js/DevelopmentFooter.jsx [hunk 3]
```diff
@@ -5,14 +5,6 @@ import revisionInfo from '../../../target/classes/io/jenkins/blueocean/revisionI
 
 export class DevelopmentFooter extends Component {
     render() {
-        if (!revisionInfo || !revisionInfo.name) {
-            var blueOceanVersion = document.getElementsByTagName('head')[0].getAttribute('data-blue-ocean-version');
```
Comment (kzantow) [inline blueocean-web/src/main/js/DevelopmentFooter.jsx]: Maybe this should have `if (!revisionInfo) { return null; }` instead?

Comment (vivek) [inline blueocean-web/src/main/js/DevelopmentFooter.jsx]: @kzantow Is there going to be a case when revisionInfo might not be present?

Comment (kzantow) [inline blueocean-web/src/main/js/DevelopmentFooter.jsx]: It's generated at build time but I've seen it missing before... there must have been some reason for the check in the first place.

Comment (vivek) [inline blueocean-web/src/main/js/DevelopmentFooter.jsx]: @kzantow Ok, I will put the condition back and return null as suggested.

**Comment by kzantow** [comment]
:bee: thanks, @vivek !

**Comment by michaelneale** [comment]
🐝

**Pull request merged**